test-clients/src/main/java/com/hedera/services/bdd/suites/SuiteRunner.java

```diff
@@ -197,9 +198,11 @@ public static void main(String... args) {
 		String[] effArgs = trueArgs(args);
 		log.info("Effective args :: " + List.of(effArgs));
 		if (Stream.of(effArgs).anyMatch("-CI"::equals)) {
-			expectedNetworkSize = EXPECTED_CI_NETWORK_SIZE;
 			var tlsOverride = overrideOrDefault(effArgs, TLS_ARG, DEFAULT_TLS_CONFIG.toString());
 			var nodeSelectorOverride = overrideOrDefault(effArgs, NODE_SELECTOR_ARG, DEFAULT_NODE_SELECTOR.toString());
+			expectedNetworkSize =  Integer.parseInt(overrideOrDefault(effArgs,
```

Comment:
A comment describing usage of `NETWORK_SIZE_ARG` and why its needed in zero stake regression will be helpful